locust/core.py

```diff
@@ -19,7 +19,7 @@
 logger = logging.getLogger(__name__)
 
 
-def task(weight=1):
+def task(weight=1, order=2147483647):
     """
```

Comment:
might be nicer to declare a MAXINT constant and use that instead of 2147483647 directly

Reply:
👍 Yep